### PR TITLE
Polish help messages

### PR DESCRIPTION
**Pull request by polina-popova**
# Please make sure all below checkboxes in the Mandatory section have been checked before submitting your PR (also don't forget to pay attention to the Depending on Circumstances section)

## Mandatory

- [x] The commit message is an imperative and starts with a capital letter (for example, `Add something`, `Remove something`, `Fix something`, etc.).
- [x] I made sure I hadn't put a period at the end of the the commit message(s).

## Depending on Circumstances

Some of the items in the section become mandatory if you are a first-time contributor and/or your changes are relevant to the items.

- [ ] Adding a new parameter I didn't forget to reflect it in `README.md`.
- [ ] Modifying a list of constants or imports I didn't brake an alphabetical order.
- [ ] (for first-time contributors) One of the commits in the pull request adds me to the **Acknowledgements** section in `AUTHORS.md`.

### CODE REVIEW

File: src/viva.js [hunk 1]
```diff
@@ -2,18 +2,14 @@
...
-//   MAXIMUM_LENGTH_OF_WAIT - The maximum number of days each request may take
+//   None
 //
 // Commands:
 //  begin group Viva Las Vegas
 //    hubot работаю из дома - sets the status of work from home and adds the corresponding event to the calendar
 //    hubot не работаю из дома - removes work from home status and deletes the corresponding event from the calendar
 //    hubot хочу в отпуск - initiates a new leave request
 //    begin admin
-//      hubot одобрить заявку @username - approves the leave request for the specified user
```
Comment (BehindLoader) [inline src/viva.js]: I don't think we need to do it. These commands are still used

File: src/viva.js [hunk 2]
```diff
@@ -2,18 +2,14 @@
 //   A Hubot script which helps users to create leave requests.
 //
 // Configuration:
-//   LEAVE_COORDINATION_CHANNEL - The channel name to handle users leave requests
-//   MAXIMUM_LENGTH_OF_LEAVE - The maximum number of days an employee is allowed to be on leave
-//   MAXIMUM_LENGTH_OF_WAIT - The maximum number of days each request may take
+//   None
```
Comment (eugulixes) [inline src/viva.js]: Why do we need to save the section? I would remove it at all.

File: src/viva.js (hunk 1 above)
Comment (polina-popova) [reply src/viva.js]: Do you mean that it is possible, that users will faced with necessity to use them except of buttons?

File: src/viva.js (hunk 2 above)
Comment (polina-popova) [reply src/viva.js]: I saved it just for unification with birthder script.

Comment (eugulixes) [reply src/viva.js]: Let's remove from both of them.

Comment (eugulixes) [reply src/viva.js]: Because it's not `None`. This is a bunch of parameters and we for some reason tell here `None`.

File: src/viva.js (hunk 1 above)
Comment (BehindLoader) [reply src/viva.js]: Yes. Of course they can

**Pull request merged**